testnets/_IBC/junotestnet-osmosistestnet.json

```diff
@@ -28,6 +28,38 @@
         "dex": "osmosis"
       }
     }
-  ]
-}
-  
\ No newline at end of file
+  ],
```

Comment:
this doesn't follow the schema, as it only supports one set of connection ids and client ids. Please use the same client+connection ids, and set up the channel using those.

Reply:
Hi JeremyParish69, 
We've created a new relayer between juno testnet(uni-6) and osmosis testnet(osmo-test-5).
how can we add our own new set of connection ids and client ids?
We can not find and use the connection id and client id that was already registered at junotestnet-osmosistestnet.json file.

Reply:
The relayer info(channel id, client id, connection id) already registered does not exists.
Is it possible for us to swap original relayer info to our new relayer info for junotestnet-osmosistestnet.json?

Reply:
I'll need to investigate further to prove it, but I believe it does indeed exist...

Reply:
I've already investigated about the existence of already registered channel.
If it was exists, we didn't need to make a new relayer between uni6 to osmo-test-5.

![image](https://github.com/cosmos/chain-registry/assets/123988921/581cee5a-44d7-462f-a9da-525cc1e7ec01)

Reply:
> I'll need to investigate further to prove it, but I believe it does indeed exist...

It does not appear to exist, which is why we created a new relayer between uni-6 and osmo-test5.  Please provide further direction so we can get this moving.  Thank you

Reply:
The original client maybe be revived via governance